contrib/core/actions/send_mail/send_mail

```diff
@@ -25,7 +25,13 @@ SEND_EMPTY_BODY=$1
 shift
 CONTENT_TYPE=$1
 shift
-BODY="$@"
+BODY="$1"
+shift
+IFS=',' read -ra ATTACHMENTS <<< "$@"
+
+if [[ $ATTACHMENTS == "None" ]]; then
```

Comment:
Yeah, if no value is provided for a particular parameter when using shell script runner + positional arguments we pass in `None` to the script for the parameter value.

The reason for that is, that in theory, your script could take 4 positional argument and let's say argument number 3 is optional. In this case we still need to pass in something (`None` in our case) if you still want to provide argument number 4 (e.g. `foo bar None arg4`).

I know that's not the best script design (you should put optional arguments at the end), but that's not always possible (e.g. multiple optional arguments).